### PR TITLE
feat(clojure-lang): add `cider-eval-sexp-at-point`

### PR DESCRIPTION
**Pull request by dijonkitchen**
Adds a useful CIDER command that doesn't have an idiomatic Spacemacs command.

Ref:
https://cider.readthedocs.io/en/latest/interactive_programming/#using-cider-mode

### CODE REVIEW

File: layers/+lang/clojure/packages.el [hunk 1]
```diff
@@ -83,6 +83,7 @@
             "eb" 'cider-eval-buffer
             "ee" 'cider-eval-last-sexp
             "ef" 'cider-eval-defun-at-point
+            "ev" 'cider-eval-sexp-at-point
```
Comment (duianto) [inline layers/+lang/clojure/packages.el]: Sort this alphabetically, like the other key bindings, that makes it easier to find which keys are used/available, without having to look through the whole list.

And add an entry for it in the clojure readme > evaluation section:
https://github.com/syl20bnr/spacemacs/tree/develop/layers/+lang/clojure#evaluation

**Comment by duianto** [comment]
Squash the commits into one.

File: layers/+lang/clojure/packages.el (hunk 1 above)
Comment (dijonkitchen) [reply layers/+lang/clojure/packages.el]: I actually tried to keep it sorted by the CIDER function so all the "eval" related ones are grouped together. Is that OK or is it better to sort by the command? 

I'll squash commits and add the docs as you note though.

**Comment by duianto** [comment]
Thank you for contributing to Spacemacs.
The PR has been cherry-picked into the `develop` branch. It's safe to delete your branch.
Commit: https://github.com/syl20bnr/spacemacs/commit/28adf374fee6739ee7ed95bbb43e3b1a68701977

I sorted it alphabetical by key binding to match the rest of the file.

**Pull request closed without merging**